tests/backup/backup_helper.go

```diff
@@ -4171,3 +4159,105 @@ func CreateRuleForBackupWithMultipleApplications(orgID string, appList []string,
 	}
 	return preRuleName, postRuleName, nil
 }
+
+// GetAllBackupNamesByOwnerID gets all backup names associated with the given ownerID
+func GetAllBackupNamesByOwnerID(ownerID string, orgID string, ctx context.Context) ([]string, error) {
+	isAdminCtx, err := portworx.IsAdminCtx(ctx)
+	if err != nil {
+		return nil, err
+	}
+	backupEnumerateReq := &api.BackupEnumerateRequest{
+		OrgId: orgID,
+		EnumerateOptions: &api.EnumerateOptions{
+			Owners: func() []string {
+				if isAdminCtx {
+					return []string{ownerID}
+				}
+				return nil
+			}(),
+		},
+	}
+	backupEnumerateResp, err := Inst().Backup.EnumerateBackup(ctx, backupEnumerateReq)
+	if err != nil {
+		return nil, err
+	}
+	backupNames := make([]string, 0)
+	for _, backupObj := range backupEnumerateResp.GetBackups() {
+		if isAdminCtx {
+			backupNames = append(backupNames, backupObj.GetName())
+		} else {
+			if backupObj.GetOwnership().GetOwner() == ownerID {
+				backupNames = append(backupNames, backupObj.GetName())
+			}
+		}
+	}
+	return backupNames, nil
+}
+
+// GetAllBackupScheduleNamesByOwnerID gets all backup schedule names associated with the given ownerID
+func GetAllBackupScheduleNamesByOwnerID(ownerID string, orgID string, ctx context.Context) ([]string, error) {
+	isAdminCtx, err := portworx.IsAdminCtx(ctx)
+	if err != nil {
+		return nil, err
+	}
+	backupScheduleEnumerateReq := &api.BackupScheduleEnumerateRequest{
+		OrgId: orgID,
+		EnumerateOptions: &api.EnumerateOptions{
+			Owners: func() []string {
+				if isAdminCtx {
```

Comment:
Schedule enumerate using ownerID is not supported. same as PB-4199

Reply:
ideally it should  be disabled for schedule as well, can you once make sure while filtering with ownerID any other user schedule is not getting listed.